types/Shipment/ServiceLevel/APC.d.ts

```diff
@@ -0,0 +1,12 @@
+export declare type ServiceLevelAPC =
```

Comment:
These are nice, though my thoughts are similar to the note I left on the predefined package section. I'm not sure defining all these constantly changing service levels and predefined packages per carrier is something we want at this stage due to complexity and maintenance cost.